zone/trading.cpp

```
@@ -2859,77 +3097,193 @@ void Client::TraderUpdateItem(const EQApplicationPacket *app)
-			e.item_unique_id        = i->GetUniqueID();
-			e.slot_id               = 0;
-			e.listing_date          = time(nullptr);
-			if (i->IsAugmented()) {
-				auto augs                 = i->GetAugmentIDs();
-				e.augment_one   = augs.at(0);
-				e.augment_two   = augs.at(1);
-				e.augment_three = augs.at(2);
-				e.augment_four  = augs.at(3);
-				e.augment_five  = augs.at(4);
-				e.augment_six   = augs.at(5);
+	std::unordered_map<std::string, TraderRepository::Trader> existing_entries_by_unique_id{};
```

Clang (14.0.6)

[warning] variable 'existing_entries_by_unique_id' is not initialized

(cppcoreguidelines-init-variables)